atlp-rwanda/eagles-bn-backend
# #175226667 create and update trip request

#### What does this PR do?
Create and update a trip request
#### Description of Task to be completed?
- create a trip request with validations
- update some trip request on requester's only allowed field
- view all trips and single trip
#### How should this be manually tested?
- clone the repository or pull changes from remote and go to `ft-trips-175226667`
- run `npm install && npm start` 
- open the documentations to check all features stated above
#### What are the relevant pivotal tracker stories?
finishes #175226667
